### PR TITLE
Update READEME for auto merging test

### PR DESCRIPTION
**Pull request by jianzhangbjz**
As title,

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,9 @@
 # QE TRT (Technical Release Team)
 
 QE TRT will make QE tasks of z-stream release more efficient and easier, improve test coverage and automate this process and release the existing release leaders' and managers' energy, and the most energy from component teams on z-stream release.
+
+### Tools
+There are two tools in this repo:
+- oar(Openshift Automatic Release), which interacte with Errata.
```
Comment (wewang58) [inline README.md:7]: oar interact with errata, jira system, google doc, slack and jenkins server

File: README.md [hunk 2]
```diff
@@ -1,3 +1,9 @@
 # QE TRT (Technical Release Team)
 
 QE TRT will make QE tasks of z-stream release more efficient and easier, improve test coverage and automate this process and release the existing release leaders' and managers' energy, and the most energy from component teams on z-stream release.
+
+### Tools
+There are two tools in this repo:
+- oar(Openshift Automatic Release), which interacte Errata, Jira, Google doc, Slack and Jenkins.
```
Comment (wewang58) [inline README.md:7]: s/interacte/interact with/
other lgtm, thanks

**Comment by jianzhangbjz** [comment]
Hi @wewang58 , could you help add `/lgtm` for it? Thanks!

**Pull request merged**